source/common/ssl/context_config_impl.cc

```diff
@@ -25,10 +25,13 @@ const std::string ContextConfigImpl::DEFAULT_CIPHER_SUITES =
     "AES256-SHA256:"
     "AES256-SHA";
 
+const std::string ContextConfigImpl::DEFAULT_ECDH_CURVES = "X25519:P-256";
```

Comment:
@mattklein123 I'm not sure what are you thoughts about changing the default. Do you prefer if I leave it as-is (i.e. P-256 only)? Should I split it into another PR?